packages/ui/src/components/editor/panels/Hierarchy/node/index.tsx

```diff
@@ -312,8 +312,14 @@ export const HierarchyTreeNode = (props: HierarchyTreeNodeProps) => {
     <li
       style={props.style}
       className={twMerge(
-        `bg-${props.index % 2 ? 'theme-surfaceInput' : 'zinc-800'}`,
-        selected ? 'border border-gray-100' : 'border-none'
+        `${
```

Comment:
do these values properly update reactively?

Reply:
It is reactive btw. But the current iteration has some other issues that aren't applying styling and have syntax errors atm.